src/components/debug-log.scss

```diff
@@ -0,0 +1,49 @@
+.debug-log {
+  position: fixed;
+  box-sizing: content-box;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #2D2D32;
+
+  overflow-y: hidden;
+  text-align: left;
+
+  margin: 0;
+  padding: 0em;
+
+  font: normal 20px Arial, Helvetica, sans-serif;
+
+  .msgs {
+    box-sizing: border-box;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    padding-left: 2em; // don't get too close to TV screen edges
```

Comment:
perhpas padding-top too?

Reply:
turns out the JS code computes top top/bottom margins to ensure that